lvgl_tft/st7789.h

```diff
@@ -110,11 +99,36 @@ extern "C"
 #define ST7789_NVMSET       0xFC    // NVM setting
 #define ST7789_PROMACT      0xFE    // Program action
 
-void st7789_init(void);
+/**
+ * Initialize the ST7789 display controller with default configuration
+ *
+ * @param drv Pointer to lv_disp_drv_t being used
+ */
+void st7789_init(lv_disp_drv_t *drv);
+
+/**
+ * Send buffer content to display  
+ *
+ * @param drv Pointer to lv_disp_drv_t being used
+ * @param area Pointer to area to be sent
+ * @param color_map Pointer to color map
+ */
 void st7789_flush(lv_disp_drv_t *drv, const lv_area_t *area, lv_color_t *color_map);
 
-void st7789_send_cmd(uint8_t cmd);
-void st7789_send_data(void *data, uint16_t length);
+/**
+ * Control display backlight
+ *
+ * @param drv Pointer to lv_disp_drv_t being used
+ * @param backlight Backlight control
+ */
+void st7789_enable_backlight(lv_disp_drv_t *drv, bool backlight);
```

Comment:
PR https://github.com/lvgl/lvgl_esp32_drivers/pull/95 carved out the backlight control from display driver.
Are you sure want to have backlight and display drivers merged together?

Reply:
Fixed with a1a64a9